### PR TITLE
Math task and its tests

### PR DESCRIPTION
**Pull request by SunM1sty**

### CODE REVIEW

File: CourseApp.Tests/CourseApp.Tests.csproj [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFramework>net5.0</TargetFramework>
```
Comment (jskonst) [inline CourseApp.Tests/CourseApp.Tests.csproj:4]: please take a look on https://github.com/ISUCT/Tprogramming_2021/pull/5/files MR to fix issue with github actions

File: CourseApp.Tests/Tests.cs [hunk 2]
```diff
@@ -0,0 +1,84 @@
+namespace CourseApp.Tests
+{
+    using Xunit;
+
+    public class Tests
+    {
+        [Fact]
```
Comment (jskonst) [inline CourseApp.Tests/Tests.cs:7]: please take a look on InlineData

File: CourseApp.Tests/Tests.cs [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+        [Fact]
+        public void TestTaskB1()
+        {
+            var a = 1.6;
+            var x = 1.28;
+            var expected = 0.6838;
+            var execution = new Calculation();
+            var actual = execution.Function(a, x);
+            Assert.Equal(expected, actual);
```
Comment (jskonst) [inline CourseApp.Tests/Tests.cs:59]: Add 3rd param - number of digits after `.`

File: CourseApp.Tests/CourseApp.Tests.csproj (hunk 1 above)
Comment (SunM1sty) [reply CourseApp.Tests/CourseApp.Tests.csproj]: fixed all troubles with project settings

File: CourseApp.Tests/Tests.cs (hunk 3 above)
Comment (SunM1sty) [reply CourseApp.Tests/Tests.cs]: Done

File: CourseApp.Tests/Tests.cs (hunk 2 above)
Comment (SunM1sty) [reply CourseApp.Tests/Tests.cs]: Done, check this out please

File: CourseApp/Calculation.cs [hunk 4]
```diff
@@ -0,0 +1,45 @@
+namespace CourseApp
+{
+    using System;
+    using System.Collections.Generic;
+    using static System.Math;
+
+    public class Calculation
+    {
+        public void TaskA(double a, double xn, double xk, double dx)
```
Comment (alexthvest) [inline CourseApp/Calculation.cs:9]: Can we move specific tasks from `Calculation` class? It should not known about them

File: CourseApp/Calculation.cs [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+            {
+                listB.Add(Function(a, x));
+            }
+
+            ConsoleOutput(listB);
+        }
+
+        public double Function(double a, double x)
+        {
+            var y = Pow(a, Pow(x, 2.0) - 1.0) - Log10(Pow(x, 2.0) - 1.0) - Pow(Pow(x, 2.0) - 1.0, 1.0 / 3.0);
```
Comment (alexthvest) [inline CourseApp/Calculation.cs:33]: Do we need this variable?

**Review by alexthvest — CHANGES_REQUESTED** [review]

File: CourseApp/Calculation.cs (hunk 5 above)
Comment (SunM1sty) [reply CourseApp/Calculation.cs]: Ofc, we don`t

File: CourseApp/Calculation.cs (hunk 4 above)
Comment (SunM1sty) [reply CourseApp/Calculation.cs]: Done,

**Review by alexthvest — APPROVED** [review]

**Comment by alexthvest** [comment]
@jskonst can we merge this?

**Comment by jskonst** [comment]
I guess so

**Pull request merged**